index.html

```diff
@@ -54,8 +55,8 @@ <h2 class="landing-row-title">Define rich, type&#8209;safe domain&nbsp;model</h2
             <div class="col-md-6 col-style-text">
                 <p class="step-title">Step 2.</p>
                 <h2>Generate the code for all tiers and multiple languages</h2>
-              <p>Java, JavaScript, and C++ are currently supported, with more languages to come soon.</p>
-            </div>
+              <p>Java, JavaScript, and C++ are currently supported. </p>
+                <!--Add a link to Roadmap for more languages/technologies support when available. --></div>
```

Comment:
Please remove the comment and have an issue.

Reply:
Removed from source code. 
Converted into issue #7 in Documentation board. 
[For some reason cannot link it to https://github.com/SpineEventEngine/documentation/issues/7]